modin/experimental/xgboost/xgboost_ray.py

```diff
@@ -94,7 +58,7 @@ def _get_dmatrix(self, *X_y):
         return xgb.DMatrix(X, y)
 
     def set_train_data(self, *X_y, add_as_eval_method=None):
-        self._dtrain = self._get_dmatrix(*X_y)
+        self._dtrain = self._get_dmatrix(list(X_y))
```

Comment:
this is probably not needed... I was actually wondering if we can pass `X_y` from outside as non-crumpled data

Reply:
fixed